src/wallet/wallet.cpp

```diff
@@ -4386,7 +4384,7 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(interfaces::Chain& chain,
     chain.loadWallet(interfaces::MakeWallet(walletInstance));
 
     // Register with the validation interface. It's ok to do this after rescan since we're still holding cs_main.
```

Comment:
I think this comment could be changed slightly now, since 'we' (the wallet) don't ever hold `cs_main` now.

`since we're still holding cs_main` -> `since we're still holding the Chain::Lock interface`.